Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'govuk_taxonomy_helpers'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage('Checkout') {
+      checkout scm
+    }
+
+    stage('Clean') {
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+    }
+
+    stage('Bundle') {
+      govuk.bundleApp()
```

Comment:
I haven't seen any other uses of it, and it looks like [we're not using it in any other ways at the moment](https://github.com/search?l=Groovy&p=4&q=org%3Aalphagov+%22bundle+install%22&type=Code).